int32 mode
float64 long_or_x
float64 lat_or_y
---
float64 long_or_x
float64 lat_or_y
